.github/workflows/pre-release_components.yml

```diff
@@ -17,13 +17,13 @@ jobs:
         run: git config --global user.email "github-actions@github.com"
 
       - name: Check out git repo
-        uses: actions/checkout@v3
```

Comment:
This is a downgrade, but this is what we have in core, and we don't / won't run this over here.